wordpress-mobile/WordPress-iOS
# Co-branding JP login screen with WP.com

Fixes: https://github.com/Automattic/jetpack-mobile-roadmap/issues/96
PT: pcdRpT-58l-p2

This PR redesigns the Jetpack app’s welcome screen to include more references to the WordPress brand, such as colours and logo.

| Before Light | After Light |
| ------  | ----- |
|![Simulator Screenshot - iPhone 15 Pro - 2023-12-27 at 15 38 18](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/b4b08000-9b60-42a3-9b07-9c7f4addf2e3)|![Simulator Screenshot - iPhone 15 Pro - 2023-12-27 at 15 32 46](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/faa05437-2427-4874-b9fa-0d09890a4e91)|
| Before Dark | After Dark |
| ------  | ----- |
|![Simulator Screenshot - iPhone 15 Pro - 2023-12-27 at 15 38 22](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/2006958a-e09a-4d62-8f45-0d731daebd08)|![Simulator Screenshot - iPhone 15 Pro - 2023-12-27 at 15 32 41](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/e54616d5-6caf-471f-bf8f-8f8209105ca0)|
## Testing Steps
Install & Launch Jetpack App and verify the colors match the designs in light and dark modes.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Only UI related changes

PR submission checklist:


UI Changes testing checklist:
